Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM ubuntu:14.04
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN locale-gen "en_US.UTF-8" "fr_FR.UTF-8" "es_MX.UTF-8" \
+RUN apt-get update \
+    && apt-get install language-pack-es -y \
+    && locale-gen "en_US.UTF-8" "fr_FR.UTF-8" "es_MX.UTF-8" \
```

Comment:
@moylop260 yes, I removed the es ones from locale-gen and left en and fr only. What do you think?